spec/models/education_spec.rb

```diff
@@ -1,5 +1,56 @@
 require 'rails_helper'
 
 RSpec.describe Education, type: :model do
-  pending "add some examples to (or delete) #{__FILE__}"
+  it 'has a valid factory' do
+    expect(FactoryGirl.build(:education)).to be_valid
+  end
+
+  describe 'converts date to strftime' do
+    education = FactoryGirl.build(:education)
+    it 'converts education start date to strftime' do
+      expect(education.pretty_date(education.start_date)).to eq '01/01/2001'
+    end
+
+    it 'converts education end date to strftime' do
+      expect(education.pretty_date(education.end_date)).to eq '05/30/2008'
+    end
+  end
+
+  describe 'test validations' do
+    it 'returns an error if start date is nil' do
+      education = FactoryGirl.build(:education, start_date: nil)
+      education.valid?
+      expect(education.errors[:start_date]).to include("can't be blank")
```

Comment:
You use the `shoulda-matcher` syntax above but not here. We should probably be consistent one way or the other